development/image-syncer/external-images.yaml

```
@@ -33,8 +33,8 @@ images:
 - source: "oryd/oathkeeper-maester:v0.1.5"
   amd64Only: true
 - source: "oryd/oathkeeper:v0.38.25-beta.1"
-- source: "postgres@sha256:4a6be41f4176af134fe8a02b9f07928fab22fb1fb2777811360154912cb682cd"
-  tag: "11.18-alpine3.17-v1"
+- source: "postgres@sha256:ee30e7ceb3588f5426fe80db5a1a4b4253c258cb9a9634680552fee5e01efb51c2b47"
```

Sorry, I copied the SHA incorrectly
```suggestion
- source: "postgres@sha256:ee30e7ceb3588f5426fe80db5a1a4b4253c258cb9a9634680552fee5e01efb51"
```

---

Fixed it already.